web/src/features/charts/hooks/usePriceChartData.ts

```diff
@@ -59,6 +62,8 @@ export function usePriceChartData() {
     markerFill,
     valueAxisLabel,
     layerStroke: undefined,
+    isPriceDisabled: Boolean(priceDisabledReason),
+    priceDisabledReason,
```

Comment:
that we're passing through two bits of data that are essentially the same instead of just passing price disabled reason then using the component to do logic.

Reply:
Ah yeah, I was also unsure about that bit - I initially leaned towards having the logic together with the other data manipulation, but I've moved it now :)